src/Aliases.ts

```diff
@@ -14,6 +14,8 @@ export default class Aliases {
     }
 
     static importAliasesFrom(shellName: string): void {
+        if (process.platform === 'win32') return;
```

Comment:
How about creating an enum with platforms in `Enums.ts` and a helper function `Utils.isWindows`?